src/components/Form/Form.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+// import { jest } from '@jest/globals';
+import { screen, render, fireEvent } from '@testing-library/react';
+// import userEvent from '@testing-library/user-event';
+
+import { Form as IMPORT, PrimaryButton } from '../../index';
+import Form from './Form';
+
+describe('Form component', () => {
+  test('can be imported from the library correctly', () => {
+    expect(IMPORT).toBeDefined();
+    expect(Form).toBeDefined();
+    expect(IMPORT).toEqual(Form);
+  });
+
+  test('can render the default component correctly', () => {
+    render(<Form onSubmit={() => {}}>
+      <button type="submit">
+        Submit
+      </button>
+    </Form>);
+    expect(screen.getByRole('button', { name: 'Submit' })).toBeInTheDocument();
+  });
+
+  test('cand render Form and submits correctly', () => {
```

Comment:
also add a test for this:

- render an input plus button inside the form,
- then focus on the input, input some input in the input,
- press enter key
- check if onSubmit was fired.